gulp-sort/gulp-sort.d.ts

```diff
@@ -34,11 +34,13 @@ declare module 'gulp-sort' {
 		 */
 		(file1: gulpUtil.File, file2: gulpUtil.File): number;
 	}
-	
+
 	/** Sort files in stream by path or any custom sort comparator */
 	function gulpSort(): NodeJS.ReadWriteStream;
 	function gulpSort(comparator: IComparatorFunction): NodeJS.ReadWriteStream;
 	function gulpSort(options: IOptions): NodeJS.ReadWriteStream;
-	
+
+	namespace gulpSort {}
```

Comment:
https://github.com/Microsoft/TypeScript/issues/5073 
If no empty namespace, It would fail when use import * as xxxx from 'xxxx'